packages/translations/src/components/localize.jsx

```diff
@@ -1,6 +1,10 @@
 import PropTypes        from 'prop-types';
 import React            from 'react';
 import { Trans }        from 'react-i18next';
+// Trans needs to have the i18n instance in scope
```

Comment:
Shouldn't this be in scope already when it's used by the consumer of `deriv-translations`? I worry this might create a separate context/scope from the one imported in the consumer itself.

Reply:
Good point, let me investigate further 👍